static/code/button/usage/basic/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Button</title>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js"></script>
+  <script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css" />
+</head>
+
+<body>
+  <ion-button>Default</ion-button>
+
+  <script>
```

Comment:
If there is another pattern that the team is familiar with, I'm open to alternatives. Since the iframe content is of a variable height, we need to resize the preview container to match the height of the frame. This doesn't happen naturally(as far as I know).

This script emits a message that the parent window can listen to, to access the height and respond to the change. If this pattern is something we move forward with, we should likely break this script out into a package/CDN link that all examples reference.